### DISCUSSION TITLE
240401

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#100] [팀회의]
## 멘토님 질문 사항

**Comment by jisunipark** [comment]
## 논의사항
- 레이아웃 안에서 모달 & 캘린더 맞추기

## 어제 한 일
- 페이지네이션 기능 완성
- ProfileStack 컴포넌트 작업
- UI 수정 및 추가

## 오늘 할 일
- BoardSection 레이아웃 맞추기
- 드래그 앤 드롭 구현
- 대시보드 페이지에서 팀 페이지 이동 시 똑같은 데이터 받아와지는 문제 해결

## 오늘의 일정 (코어 타임)
- 12:00 - 12:30 온라인 일정

## 멘토님 질문 사항

**Comment by MoonPillGyeom** [comment]
## 논의사항
- 채빈님 예전에 초반에 모달위에 모달? 무슨 말씀 하셨던것 같은데 그거 나중에 여쭤보러 가겠습니다
- 
## 어제 한 일
- 대충 급한 에러처리..했습니다
- 기타 등등 했는데 잘 기억이 안납니다

## 오늘 할 일
- members만들기

## 오늘의 일정 (코어 타임)
 - 14:30 ~ 매니저상담

## 멘토님 질문 사항

**Comment by caseV12** [comment]
## 논의사항
필겸님 모달 레이아웃. 목표 기한.
## 어제 한 일
myPage '히스토리 관리 모달' 부분 빼고 ui, 로직 완성
useUserContext, privateRoutes 로직 계속 변경 >결론 user state랑 dispatch를 컨텍스트 분리하는 변경 필요할 것 같음
## 오늘 할 일
모달 close 이벤트가 정상 작동을 안함 formMethod="dialog"해도
히스토리 관리 모달
members 건드리기
context 건드리기
## 오늘의 일정 (코어 타임)

## 멘토님 질문 사항
